twine/commands/upload.py

```diff
@@ -60,6 +60,12 @@ def upload(upload_settings: settings.Settings, dists: List[str]) -> None:
     repository_url = cast(str, upload_settings.repository_config["repository"])
 
     print(f"Uploading distributions to {repository_url}")
+    if upload_settings.verbose:
+        for dist_to_upload in uploads:
+            # If the file size is <.1 MB we display in KB
+            file_size, size_unit = utils.get_file_size(dist_to_upload)
+            print(f"   {dist_to_upload} ({file_size} {size_unit})")
```

Comment:
If this is a float, how will this get formatted? What's the accuracy? How many significant digits? We should be consistent here by using format specifiers

Reply:
The output would look like this: 
<img width="922" alt="Screen Shot 2020-06-08 at 2 19 31 PM" src="https://user-images.githubusercontent.com/35483161/84085487-8afb3c80-a99a-11ea-9c07-d5aacbb38117.png">
I determined the number of decimal points to round to(1) by copying how PyPI displays the size of files on their website. The function get_file_size in utils.py determines the unit(KB or MB) returns the size rounded to one decimal place as well as the unit. 

I will add the format specifiers to the string. Additionally, it seems that get_file_size is causing codecov tests to fail. The test I thought of adding to pass the codecov would check that the size of one of the helper dists is accurate and outputs the correct unit. Are there any additional tests you would like to see? Thanks for the help.

Reply:
W.r.t format specifiers, I think what @sigmavirus24 is saying is that it's probably best not to directly round in the utils function, but to instead print the returned float rounded to a certain number of decimal places using something like the following:

`print(f"   {dist_to_upload} ({file_size:.1f} {size_unit})")`

As an example, `print(f"{2.71828:.2f}")` will automatically round and print `2.72`.